source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2140,6 +2144,8 @@ export class Image extends CardElement {
                 imageElement.style.backgroundColor = backgroundColor;
             }
 
+            this.configureImageForForceLoading();
+
             imageElement.src = <string>this.preProcessPropertyValue(Image.urlProperty);
```

Comment:
Does `Image.urlProperty` hold the same value as `this.url`?

Reply:
No. This is really confusing. the property does a few things. it serializes and deserializes. it also serves as a key to a property bag of Adaptive element. The property bag holds the value for the url. 
We can make it more clear by using named parameter for preProcessPropertyValue as it takes two parameters, prop which is key that used here and propertyValue, intended value, but the preProcessPropertyValue is used in many places other than Image, so it's out of scope for this PR request.